### ISSUE TITLE
After login into account on localhost cms change the last_ip account column to unsuported core value that lead it to crash

### ISSUE THREAD

**Issue by CraftedRO** [FusionGen/FusionGEN#212]
After login into account on localhost cms change the last_ip account column to unsuported core value (::1) that lead it to crash when using .pinfo on a character from that account if account is banned

https://github.com/TrinityCore/TrinityCore/issues/28911

> Shauren:
> We only support ipv4 addresses in that column (and worldserver only writes ipv4 there)

**Comment by CraftedRO** [comment]
I tested now and old fusion doesn't change last_ip data.

**Comment by CraftedRO** [comment]
I tested now and old fusion doesn't change last_ip data upon login into cms.

**Comment by CraftedRO** [comment]
yea and that's the correct behaviour I guess

**Comment by CraftedRO** [comment]
well just wanted you to know about it (will probably be more reports about this in the future) but I still think that account > last_ip table shoud be left for ingame logins, the cms if it need shoud have separate table or otherwise valid ipvp format since there's no other option.

**Comment by CraftedRO** [comment]
well just wanted you to know about it (will probably be more reports about this in the future) but I still think that account > last_ip table shoud be left for ingame logins, the cms if it need shoud have separate table or otherwise valid ipv4 format since there's no other option.

**Comment by Err0r1** [comment]
::1 is valid
Trinitycore just can't handle it because they don't support ipv6 (in 2023..)

The cms is allowed to update/add stuff in the auth database, but not the column? I don't quite understand that

**Comment by CraftedRO** [comment]
most likely azerothcore or probably others can't too so if we handle those emulators we should handle them completely as they are now idk

**Comment by Err0r1** [comment]
What about the master branch? Doesn't it support IPv6?
Where do I make fun of the project?
In 2023, IPv6 is something completely normal and widespread

**Comment by Shauren** [comment]
Yes, master branch client supports ipv6, master branch does not (yet)

As for ipv6 being widespread? i dont get a v6 address from my isp so not as widespread as you might expect (and no, its not some trash tier connection)

**Comment by Err0r1** [comment]
Maybe it's a Germany or EU thing
Here many have an IPv6 and only on request you get an ipv4 (some still offer ipv4 by default)

I'll think about how to solve this

**Comment by CraftedRO** [comment]
I somehow get both ip versions from my isp kek :rofl:

**Comment by CraftedRO** [comment]
ok seems TC fixed invalid ip crash so it is up to you to decide if you still change this for other emulators or no idk

**Comment by Racknar0** [comment]
The problem is when someone is logged into the game, and logs into the web, the last ip is changed to v6, but one solution is to put a pseudiIP v4 from cloudflare, that's how I solved it

Srry for the English.

**Comment by Err0r1** [comment]
I will change it soon
Stuff related to CMS stays in CMS
